scout/server/blueprints/alignviewers/templates/alignviewers/igv_sashimi_viewer.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta content="Jim Robinson" name="author">
+    <link rel="shortcut icon" href="//igv.org/web/img/favicon.ico">
+    <title>igv.js</title>
+
+    <!-- IGV JS -->
+    <script src="https://cdn.jsdelivr.net/npm/igv@2.7.5/dist/igv.min.js"></script>
```

Comment:
What do you say, is it starting to get time for a constant for this perhaps? I wouldn't mind that for all versioned js we have actually - they are not too many, but hunting around to update them is a little silly.

Reply:
Sounds like a good idea, we could import it from somewhere!

Reply:
Yes, like a constants file where we have URLs anyway? Seems like a Good Thing! Could be a separate PR if you tire of it.

Reply:
We can have it in here, I'm going to put it in the same file if the the igv tracks

Reply:
In the end it was easier to create a macro in a new utils.html file and have it there